helper/ssh/ssh.go

```diff
@@ -42,3 +51,62 @@ func FileSigner(path string) (ssh.Signer, error) {
 
 	return signer, nil
 }
+
+// FileSigner returns an ssh.Signer for a key file.
+func FileSignerWithCert(path string, certificatePath string) (ssh.Signer, error) {
+
+	keySigner, err := FileSigner(path)
+
+	if err != nil {
+		return nil, err
+	}
+
+	if certificatePath == "" {
+		return keySigner, err
+	}
+
+	// Load private key
+	if err != nil {
```

Comment:
Where's this error coming from? Looks like you may have missed a step or copypasted the error check twice.

Reply:
Yeah I fixed - thanks for the catch.